cabal.project

```diff
@@ -64,6 +64,7 @@ packages:
   libs/cardano-ledger-test
   libs/plutus-preprocessor
   libs/ledger-state
+  libs/tasty-compat
```

Comment:
Sorry, @neilmayhew, I am really against adding another package to our release process just to provide this temporary compatibility.

I can suggest moving this to cardano-ledgder-core:testlib instead and at some point we can just get rid of it, once tasty becomes a thing of the past.

Reply:
I hadn't thought about the release process. Yes, a new library does add a lot of unnecessary overhead. Putting it in `cardano-ledger-core:testlib` will be much easier. Also, since the library is providing only what's needed by ledger it seems better to have it in a ledger-specific library.

Reply:
Moved to a sublib of `cardano-ledger-core`. However, the packages that are dependencies of `cardano-ledger-core` can't use it (because it would introduce circular dependencies) so they have had to be converted explicitly.